pkg/commands/image/list.go

```diff
@@ -47,18 +61,27 @@ The namespace defaults to the kubernetes current-context namespace.`,
 		SilenceUsage: true,
 	}
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", "", "kubernetes namespace")
+	cmd.Flags().BoolVarP(&allNamespaces, "all-namespaces", "A", false, "Return objects found in all namespaces")
+	cmd.Flags().StringArrayVar(&filters, "filter", nil,
+		`Each new filter argument requires an additoinal filter flag.
+Multiple values can be provided using comma separation.
+Supported filters and values:
+builder=string
+clusterbuilder=string
+latest-reason=commit, trigger, config, stack, buildpack
+ready=true, false, unknown`)
```

Comment:
small thing again... a space indent on the supported filter/value lines make the help text more readable at a glance 🤷‍♂️ 
![Screen Shot 2021-01-08 at 2 50 48 PM](https://user-images.githubusercontent.com/1648613/104070850-d64a2780-51d5-11eb-81b1-c88054bcbee2.png)
![Screen Shot 2021-01-08 at 2 50 19 PM](https://user-images.githubusercontent.com/1648613/104070857-dba77200-51d5-11eb-9d29-cd586dd61040.png)